app/main.py

```diff
@@ -2,5 +2,18 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    def wrapper(*args) -> Callable:
+        key = (func.__name__,) + args
+
```

Comment:
- Good to add an annotation from `functools` module
- Also, it should work with `**kwargs`. Create a unique key using args and kwargs